### ISSUE TITLE
Silently fails when python-lzma is not installed 

### ISSUE THREAD

**Issue by Fuzion24** [rickysarraf/apt-offline#52]
It wasn't apparent that I didn't have python-lzma installed without having to add logging to figure out it was failing here because the module import was failing: 

https://github.com/rickysarraf/apt-offline/blob/master/apt_offline_core/AptOfflineLib.py#L459

May I suggest adding a logging message : "is python-lzma installed?  pip install lzma  // apt-get install python-lzma"

**Comment by rickysarraf** [comment]
```
2016-07-03 / 12:26:51 ♒♒♒  ☺  
rrs@learner:~/.rrs-home/devel/apt-offline/apt-offline (master)$ apt-cache show apt-offline
Package: apt-offline
Version: 1.7
Installed-Size: 304
Maintainer: Ritesh Raj Sarraf <rrs@debian.org>
Architecture: all
Depends: python, python:any (<< 2.8), python:any (>= 2.7.5-5~), python-argparse, apt, less, python-magic
Recommends: python-soappy, debian-archive-keyring, python-lzma
Description-en: offline APT package manager
```

If you use Debian package, then it should get pulled in as a Recommends. But yes, I agree a message will be better. But probably in AptOfflineCore. I'll fix it soon. Thanks.

**Comment by rickysarraf** [comment]
I was hit by this bug back in February. Sorry for the delay.

**Comment by staticfrost** [comment]
This is still an issue for me on 1.8.1 Ubuntu 16.04 fresh server.

**Comment by rickysarraf** [comment]
> This is still an issue for me on 1.8.1 Ubuntu 16.04 fresh server.

Yes. This was fixed after 1.8.1.
Looks like I've procrastinated way too much in desire to fix one (user) security bug/feature elsewhere. I guess I should make a release first. Thank you for reminding. :-)
